Extras/BulletRoboticsGUI/CMakeLists.txt

```diff
@@ -107,6 +109,7 @@ SET(BulletRoboticsGUI_SRCS ${BulletRoboticsGUI_INCLUDES}
 	../../examples/SharedMemory/PhysicsServerExample.cpp
 	../../examples/SharedMemory/PhysicsServerExampleBullet2.cpp
 	../../examples/SharedMemory/SharedMemoryInProcessPhysicsC_API.cpp		
+    ../../examples/RobotSimulator/b3RobotSimulatorClientAPI.cpp
```

Comment:
Fix tab style please

Reply:
Done in 5bd43c5d0a0858bfd92e3a4045f550b345256490